ansible/callback_plugins/vault_cache.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+from ansible.plugins.callback import CallbackBase
+from ansible.utils.display import Display
+
+display = Display()
+
+class CallbackModule(CallbackBase):
+    CALLBACK_VERSION = 2.0
+    CALLBACK_TYPE = 'aggregate'
+    CALLBACK_NAME = 'cache_cleanup'
+
+    def __init__(self):
+        super(CallbackModule, self).__init__()
+        self.cache_dir = "./ansible/files/cache"
```

Comment:
How about a bit more explicit?
```suggestion
        self.cache_dir = "./ansible/files/cache/vault"
```